shell/platform/common/client_wrapper/include/flutter/binary_messenger.h

```diff
@@ -7,6 +7,8 @@
 
 #include <functional>
 #include <string>
+#include <cstdint>
```

Comment:
I think the linter will complain that this needs to be in alphabetical order

Reply:
good point